cmd/snap/cmd_alias.go

```diff
@@ -65,6 +69,42 @@ func (x *cmdAlias) Execute(args []string) error {
 		return err
 	}
 
-	_, err = wait(cli, id)
+	chg, err := wait(cli, id)
+	if err == nil {
+		if err := showAliasChanges(chg); err != nil {
+			return err
+		}
+	}
+
 	return err
 }
+
+type changedAlias struct {
+	Name   string `json:"name"`
+	Target string `json:"target"`
+}
+
+func showAliasChanges(chg *client.Change) error {
+	var added, removed []*changedAlias
+	if err := chg.Get("aliases-added", &added); err != nil && err != client.ErrNoData {
+		return err
+	}
+	if err := chg.Get("aliases-removed", &removed); err != nil && err != client.ErrNoData {
+		return err
+	}
+	w := tabwriter.NewWriter(Stdout, 2, 2, 1, ' ', 0)
+	if len(added) != 0 {
+		fmt.Fprintf(w, i18n.G("Added:\n"))
+		for _, ta := range added {
+			fmt.Fprintf(w, "\t- %s => %s\n", ta.Name, ta.Target)
```

Comment:
This is slightly different from what's in the PR description. I prefer this indentation (2) and ordering (alias => snap.app) too, rather than the one in the description.